patches/asmpatchhandler.py

```diff
@@ -39,7 +39,7 @@
 # Adds a patch to nnSdk to route all vfprintf calls to the debug output
 # These will be printed to the console on yuzu
 # These prints will spam the console so don't leave this set to True
-ASM_DEBUG_PRINT = False
+ASM_DEBUG_PRINT = True
```

Comment:
Same as with #469 Is leaving this as True intentional or not?

Reply:
Explained here: https://github.com/mint-choc-chip-skyblade/sshd-rando/pull/469#discussion_r2035277406

I've updated the comment to better describe how things currently work :p